bionic/task_state.py

```diff
@@ -344,6 +341,7 @@ def initialize(self, bootstrap, flow_instance_uuid):
         # Lastly, set up cache accessors.
         if self.should_persist:
             if bootstrap is None:
+                # TODO TaskKey.entity_name does not exist, so this will crash.
```

Comment:
Why don't you fix this too? Looks like we use this to identify the entity in error log.

Reply:
Oh I see that it's addressed in a follow up PR #141